src/components/preview/PreviewThumbnail.tsx

```diff
@@ -1,76 +1,147 @@
 import React from 'react';
 import classNames from 'classnames';
+import { useNavigate } from 'react-router-dom';
 import Bookmark from '../../assets/icons/bookmark.svg?react';
+import UnclickBookmark from '../../assets/icons/unclickBookmark.svg?react';
+import DefaultImage from '../../assets/icons/defaultImage.svg?react';
 import styles from './previewThumbnail.module.scss';
 
 interface PreviewThumbnailProps {
+  ideaId: number;
   imageUrl?: string;
   profileImage?: string;
   username?: string;
   description?: string;
   price?: number;
   isBookmarked?: boolean;
+  auth?: 'ALL' | 'COMPANY' | 'ME';
+  category?: string;
+  saves?: number;
+  views?: number;
+  size?: 'normal' | 'large';
   onBookmarkClick?: () => void;
```

Comment:
갖고 오는 props가 너무 많아서 하나의 개체로된 prop 하나로 받아오시는 건 어떠신가요?
```suggestion
data: PreviewTumbnailType = {
  ideaId: number;
  imageUrl?: string;
  profileImage?: string;
  username?: string;
  description?: string;
  price?: number;
  isBookmarked?: boolean;
  auth?: 'ALL' | 'COMPANY' | 'ME';
  category?: string;
  saves?: number;
  views?: number;
  size?: 'normal' | 'large';
  onBookmarkClick?: () => void;
 }
```